FetchOpenWeatherMapMeasures.csx

```diff
@@ -16,7 +16,7 @@ using InfluxDB.Net.Infrastructure.Influx;
 using InfluxDB.Net.Infrastructure.Configuration;
 
 private const string _apiBaseAddress = @"http://api.openweathermap.org/data/3.0/measurements";
-private const string _databaseName = "WeatherSensorMessurements";
+private const string _databaseName = "WeatherSensorMeasurements";
```

Comment:
Not sure about this, probably breaks your grafana dashboard and datasource config 😞 
From grafana/datasoures for example:

`apiVersion: 1

datasources:
  - name: WeatherSensorMessurements
    type: influxdb
    access: proxy
    database: WeatherSensorMessurements
    user: root
    password: root
    url: http://database:8086`

Reply:
Just update the dashboard and datasource files as well, no problem starting from scratch since the data is stored elsewhere.